src/Common/tests/Tests/System/Net/Http2/HPackIntegerTest.cs

```diff
@@ -34,26 +33,30 @@ public void HPack_IntegerEncode_ShortBuffer(int value, int bits, byte[] expected
             Assert.False(success);
         }
 
-        [Theory]
-        [MemberData(nameof(IntegerCodecRoundTripSamples))]
-        public void HPack_IntegerRoundTrip(int value, int bits)
+        [Fact]
+        public void IntegerEncoderDecoderRoundtrips()
```

Comment:
tests with tens of thousands of member data entries cause test discovery performance problems in VS. We're still getting the same coverage here, I've just internalyzed it into the test.